questiontwo

```diff
@@ -0,0 +1,16 @@
+function frequencyCounter(character) {
+  var input = "asdwqdaabbwwddssafhjkls";
+  var frequency = {}; 
+  for (var index = 0; index < 23; index++) {
+    var char = input[index];     
+    if (frequency[char] == undefined) {
```

Comment:
@JainVayam Instead of using 'frequency[char] == undefined', you can use if (!frequency[char]).